### PR TITLE
Set Chromium version for api.MediaDeviceInfo.toJSON

### PR DESCRIPTION
**Pull request by vinyldarkscratch**
This PR uses the mdn-bcd-collector to determine the version number for `api.MediaDeviceInfo.toJSON`.

### CODE REVIEW

File: api/MediaDeviceInfo.json [hunk 1]
```diff
@@ -25,11 +25,11 @@
             "version_added": false
           },
           "opera": {
-            "version_added": false,
+            "version_added": "42",
             "notes": "This interface can be used in Opera by using the <a href='https://github.com/webrtc/adapter'>adapter.js</a> polyfill."
```
Comment (foolip) [inline api/MediaDeviceInfo.json:29]: Let's delete this note now that it is supported.

File: api/MediaDeviceInfo.json [hunk 2]
```diff
@@ -295,11 +295,11 @@
               "version_added": false
             },
             "opera": {
-              "version_added": false,
+              "version_added": "42",
               "notes": "For earlier versions, this method is available through the <a href='https://github.com/webrtc/adapter'>adapter.js</a> polyfill"
```
Comment (foolip) [inline api/MediaDeviceInfo.json:299]: Let's delete this note too.

**Review by foolip — COMMENTED** [review]
LGTM % deleting stale notes.

**Comment by foolip** [comment]
Ping @vinyldarkscratch

File: api/MediaDeviceInfo.json (hunk 1 above)
Comment (vinyldarkscratch) [reply api/MediaDeviceInfo.json]: These changes weren't supposed to be in this PR, oops!  Cherry-picking.

**Review by foolip — APPROVED** [review]

**Pull request merged**